src/App.vue

```diff
@@ -1,27 +1,28 @@
 <template>
-  <img alt="Vue logo" src="./assets/logo.png">
+  <!-- <img alt="Vue logo" src="./assets/logo.png"> -->
   <!-- <HelloWorld msg="Welcome to Your Vue.js App" /> -->
-  <!-- <header class="header" v-bind:title="title"></header> -->
-  <Footer class="footer" v-bind:message="footerMessage"></Footer>
+  <Header class="header" v-bind:title="title"></Header>
   <banner class="banner" v-bind:bannerMessage="messageToDisplay" v-bind:bannerType="messageType"></banner>
-  <weather>
+  <Search class="weather-search" v-on:search-city="searchCity"></Search>
+  <weather class="weather-results" v-bind="weatherData" v-if="validWeatherData" v-on:clear-weather-data="resetData">
   </weather>
-  <Search></Search>
+  <Footer class="footer" v-bind:message="footerMessage"></Footer>
 </template>
 
 <script>
 // import HelloWorld from './components/HelloWorld.vue';
```

Comment:
can you pls remove all unwanted commented code if not needed

Reply:
Please go through all files and remove unwanted comments, I have not commented every places but you can take care and check all places